src/main/webapp/js/index.js

```diff
@@ -44,11 +44,29 @@ function displayReceipts(label, receipts) {
   }
 }
 
+/**
+ * Creates error message based on existing HTML template.
+ * @param {string} label User-entered label.
+ */
+function createErrorMessageElement(label) {
+  // Clone error message from template.
+  const errorMessageClone =
+      document.querySelector('#error-message-template').content.cloneNode(true);
+
+  // Fill in template fields with correct information.
+  errorMessageClone.querySelector('h3').innerText =
+      'Sorry, no results found for "' + label +
+      '". Please try your search again or try a different query.';
```

Comment:
Embedding the label in the string using template literals would make this easier to read.